### DISCUSSION TITLE
IFEO - How to change calling arguments of process without messing anything for it

### DISCUSSION THREAD

**Discussion by PeterHevesi** [winsiderss/systeminformer#977] [Questions / Answers] [answered]
Hi gyus,
I know, that I shouldn't ask this question here, but many of you guys here probably knows IFEO very well, because it is used in ProcessHacker, so you might be able to help me.

I already created this question on StackOverflow, where I explained it in detail:
https://stackoverflow.com/questions/68982955/ifeo-how-to-change-calling-arguments-of-process-without-messing-anything-for-i

If any of you guys help me, I would be extremely happy and grateful :)

**Comment by dmex** [comment]
Do you know what might cause this issue @diversenok ?

**Comment by PeterHevesi** [comment, marked as answer]
He already replied to me here, I haven't notice :D So I will try to solve it now by myself using his recommendations :) And I will update with my solution once I do it :)
https://github.com/diversenok/ExecutionMaster/issues/3
